src/com/lftechnology/java/training/srijan/basics/coreapp/controller/EmployeeManagementController.java

```diff
@@ -0,0 +1,307 @@
+	 * @author srijan
+	 */
+	public void normalUserOptionHandler(int normalUserChoice, String userName) {
+
+		switch (normalUserChoice) {
+		case 1:
+			employees = employeeServiceImpl.viewAllUser(e);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		case 2:
+			employee = employeeServiceImpl.findUserByUserName(userName);
+			LOGGER.log(Level.INFO, "{0}", employee);
+			Employee changedProfileDetails = changeProfile(employee);
+			employeeServiceImpl.updateProfile(changedProfileDetails);
+			break;
+		default:
+			LOGGER.log(
+				Level.INFO,
+				"Please choose option properly,Default view all user block");
+			employees = employeeServiceImpl.viewAllUser(e);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		}
+	}
+
+	/**
+	 * <p>edit and terminate user</p>
+	 * 
+	 * @return
+	 * @author srijan
+	 */
+	public int editAndTerminateEmployee() {
+
+		Scanner input = null;
+		int employeeIdToEdit = 0;
+		try {
+			input = new Scanner(System.in);
+			employeeIdToEdit = input.nextInt();
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		finally {
+			input.close();
+		}
+		return employeeIdToEdit;
+	}
+
+	/**
+	 * <p>Change profile of an user</p>
+	 * 
+	 * @param emp
+	 * @return
+	 * @author srijan
+	 */
+	public Employee changeProfile(Employee emp) {
+
+		Scanner input = null;
+		try {
+			input = new Scanner(System.in);
+			LOGGER.log(Level.INFO, "Enter Username:");
+			this.username = input.nextLine();
+			if (!"".equals(username))
+				emp.setUserName(username);
+			LOGGER.log(Level.INFO, "Enter Fullname:");
+			this.fullname = input.nextLine();
+			if (!"".equals(fullname))
+				emp.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department:");
+			this.department = input.nextLine();
+			if (!"".equals(department))
+				emp.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			this.address = input.nextLine();
+			if (!"".equals(address))
+				emp.setAddress(address);
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		finally {
+			input.close();
+		}
+		return emp;
+	}
+
+	/**
+	 * <p>Takes input from user the search choice and value of choice</p>
+	 * 
+	 * @return
+	 */
+	public SearchEmployee choiceOfSearch() {
+
+		SearchEmployee emp = new SearchEmployee();
+		Scanner inputChoice = null;
+		Scanner input = null;
+		String choiceValue = "";
+		int choice = 0;
+		try {
+			inputChoice = new Scanner(System.in);
+			input = new Scanner(System.in);
+			choice = inputChoice.nextInt();
+			if (choice == 1)
+				emp.setSearchChoice("department");
+			else if (choice == 2)
+				emp.setSearchChoice("fullname");
+			else if (choice == 3)
+				emp.setSearchChoice("address");
+			else
+				emp.setSearchChoice("default");
+
+			LOGGER.log(Level.INFO, "Write {0}?", emp.getSearchChoice());
+			choiceValue = inputChoice.nextLine();
+			if (!"".equals(choiceValue)) {
+				emp.setChoiceValue(choiceValue);
+				System.out.println("here");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)